build.gradle.kts

```diff
@@ -20,6 +22,7 @@ dependencies {
 
     testImplementation("com.github.gumtreediff", "gen.python", "2.1.2")
     testImplementation(kotlin("test-junit"))
+    dokkaHtmlPlugin("org.jetbrains.dokka:kotlin-as-java-plugin:1.4.0-rc")
```

Comment:
You need this line only if you want to generate html docs which I'm not sure was your intention, right?

Reply:
sry, I was exploring dokka possibilities and didn't cleaned up after that.